src/extensions/web/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/RazorMappedTextReplacer.cs

```diff
@@ -70,14 +70,7 @@ public void ApplyTextReplacements(IEnumerable<MappedTextReplacement> replacement
                     var originalText = replacement.OriginalText.ToString().TrimStart();
                     var updatedText = replacement.NewText.ToString().TrimStart();
 
-                    // Generally, it's not necessary to minimize replacements since the text should be the same in both documents
-                    // However, in the specific case of text being added before the first #line pragma in a Razor doc, it's possible
-                    // that C# unrelated to the original document will show up with the replaced text, so minimize replacements only
-                    // in the case that text is being inserted before the original first line.
-                    if (replacement.StartingLine == 0)
```

Comment:
Yep, this isn't needed anymore because the improved MinimizeReplacement method can run in all situations. Previously, I didn't want to minimize too liberally because very small changes would get confusing. Now that the 'smallest' a replacement can be minimized to is a whole word, it seems more ok to use that in all situations.